bindings/node/test/autoEncrypter.test.js

```diff
@@ -143,6 +143,27 @@ describe('AutoEncrypter', function () {
       });
     });
 
+    it('should decrypt mock data and mark decrypted items if enabled for testing', function (done) {
+      const input = readExtendedJsonToBuffer(`${__dirname}/data/encrypted-document.json`);
```

Comment:
Should/can we add a test for the js object case so that we cover both top level code paths (object and buffer)? 

It could be also good to cover the new error case (though I don't know if it's too hard to do to be worthwhile).

Does the test document cover both parts of the `if (!decrypted || typeof decrypted !== 'object') return` condition and also the case of the Binary with sub_type other than 6?

Reply:
> Should/can we add a test for the js object case so that we cover both top level code paths (object and buffer)?

Sure, I think that’s fine. Done! :+1: 

> It could be also good to cover the new error case (though I don't know if it's too hard to do to be worthwhile).

Yeah, it would be hard – it’s more of an assertion than a real error case, and triggering it would require mocking out other code to make it appear buggy.

> Does the test document cover both parts of the `if (!decrypted || typeof decrypted !== 'object') return` condition 

Not yet, I think, but it’s easy to add.

> and also the case of the Binary with sub_type other than 6?

Also easy to add, done :+1: